server/src/main/java/org/opensearch/client/ClusterAdminClient.java

```diff
@@ -791,4 +793,14 @@ public interface ClusterAdminClient extends OpenSearchClient {
      * Delete specified dangling indices.
      */
     ActionFuture<AcknowledgedResponse> deleteDanglingIndex(DeleteDanglingIndexRequest request);
+
```

Comment:
Should be under `AbstractClient`, it does not belong to admin functionaliy (I think), should be treated the same way as `create` PIT

Reply:
All "cluster:admin" actions are grouped under "cluster admin client" . Since we moved the action to "cluster:admin" , made this change.